src/services/schema-file-updater.js

```diff
@@ -1,31 +1,34 @@
 const _ = require('lodash');
 const fs = require('fs');
-const logger = require('../services/logger');
 const { parameterize } = require('../utils/string');
 const { prettyPrint } = require('../utils/json');
 
-function SchemaFileUpdater(filename, collections, meta, serializerOptions) {
-  const formatObject = (object, attributes) => {
-    const objectFormated = {};
-    const objectOrdered = _.sortBy(
-      Object.keys(object),
-      (attribute) => attributes.indexOf(attribute),
-    );
-    _.each(objectOrdered, (attribute) => {
-      if (attributes.includes(attribute)) {
-        objectFormated[attribute] = object[attribute];
-      }
-    });
-    return objectFormated;
-  };
-
-  function setDefaultValueIfNecessary(object, property, value) {
-    if (!Object.prototype.hasOwnProperty.call(object, property)) {
-      object[property] = value;
+function formatObject(object, attributes) {
+  const objectFormated = {};
+  const objectOrdered = _.sortBy(
+    Object.keys(object),
+    (attribute) => attributes.indexOf(attribute),
+  );
+  _.each(objectOrdered, (attribute) => {
+    if (attributes.includes(attribute)) {
+      objectFormated[attribute] = object[attribute];
     }
+  });
+  return objectFormated;
+}
+
+function setDefaultValueIfNecessary(object, property, value) {
+  if (!Object.prototype.hasOwnProperty.call(object, property)) {
+    object[property] = value;
   }
+}
 
-  const cleanFields = (fields) => {
+class SchemaFileUpdater {
+  constructor({ logger }) {
+    this.logger = logger;
+  }
```

Comment:
Yeah 👍 this is good.
Please note: `formatObject` and `setDefaultValueIfNecessary` should be inside the class (or inside another class if you consider it's another concern) because these functions are hidden, they are can't be tested unitary.

Reply:
Thank you. It has been fixed here: https://github.com/ForestAdmin/forest-express/pull/528/commits/cfa930dd857c5bce793b3a1ca48742dd990b3c58